clang/test/AST/Interp/constexpr-frame-describe.cpp

```diff
@@ -0,0 +1,87 @@
+// RUN: %clang_cc1 -DNEW_CONST_INTERP -std=c++20 -fexperimental-new-constant-interpreter -fsyntax-only -verify %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -std=c++20 -fexperimental-new-constant-interpreter -fsyntax-only -verify %s
```

add another RUN line for the current interpreter and do the `-verify=ref,both` and `-verify=expected-both` dance. You can then uncomment the non-working example and add expected output for `ref-` and `expected-` separately.

Reply:
Done!